multi-agent/lib/mas/elements/sandboxes/openshell_sandbox/openshell_sandbox_factory.py

```
@@ -0,0 +1,30 @@
+                endpoint=cfg.gateway_url,
+                ca_pem=cfg.ca_cert,
+                cert_pem=cfg.tls_cert,
+                key_pem=cfg.tls_key,
+                keep_sandbox=cfg.keep_sandbox,
+                workdir=cfg.workdir,
+            )
+        except Exception as e:
+            raise PluginConfigurationError(
+                f"OpenShellSandboxFactory.create() failed: {e}",
+                cfg.model_dump(),
+            ) from e
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)